java/test/jmri/jmrix/tmcc/SerialThrottleTest.java

```diff
@@ -270,13 +270,73 @@ public void testSetF21() {
         boolean f21 = false;
         instance.setF21(f21);
     }
-    
+
+    /**
+     * Test of setF22 method, of class AbstractThrottle.
+     */
+    @Test
+    @Override
+    public void testSetF22() {
+        boolean f22 = false;
+        instance.setF21(f22);
```

Comment:
Is this correct? `setF21` and `f22` ?

Reply:
Thanks! Fixed.  The tests themselves aren't very specific, and didn't catch that.